app/routes/questionnaire.py

```diff
@@ -295,6 +295,10 @@ def get_thank_you(schema, session_store):
                     email=confirmation_email.get_url_safe_serialized_email(),
                 )
             )
+        else:
```

Comment:
The `else` is not required - if the previous `if` statement matches then the method returns.